cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java

```diff
@@ -844,4 +858,36 @@ public void testDeleteMethodDeleteOneAgentConfigurationParameter() throws Except
     
     } // testDeleteMethodDeleteOneAgentConfigurationParameter
     
+    /**
+     * [ManagementInterface] -------- 'Agent configuration file name starts with "agent_"'.
+     * @throws java.lang.Exception
+     */
+    @Test
+    public void testAgentBadConfigurationFileName() throws Exception {
+        System.out.println(getTestTraceHead("[ManagementInterface.handleAgentConfFileName]") +
+                  " - Agent configuration doesn't start with 'agent_'.");
+        
+        StatusExposingServletResponse responseWrapper = new StatusExposingServletResponse(response);
+        ManagementInterface managementInterface = new ManagementInterface(new File(""), null, null, null, 8081, 8082);
+        
+        
+         try {		
+             managementInterface.handleDeleteAdminConfigurationAgent(mockRequestBadFileName, responseWrapper, false);
+         } catch (Exception x) {		
+             System.out.println("There was some problem when handling the request");		
+             throw x;		
+         } // try catch
+         
+         try {		
+             assertEquals(HttpServletResponse.SC_BAD_REQUEST, responseWrapper.getStatus());		
+             System.out.println(getTestTraceHead("[ManagementInterface.handleAgentConfFileName]") + " - "
+                     + "OK  - An agent configuration file not starting with 'agent_' has been detected");		
+         } catch (AssertionError e) {		
+             System.out.println(getTestTraceHead("[ManagementInterface.handleAgentConfFileName]") + " - "
+                     + "FAIL - There are some problems with your request");		
```

Comment:
This message must be something like "An agent configuration file not starting with 'agent_' has not been detected".